src/routes/category.router.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const router = express.Router();
+
+const CategoryController = require("../controllers/category.controller");
+const { verifyAuth } = require("../middlewares/verifyAuth");
```

Comment:
Where you have used `verifyAuth` in this file?

Reply:
Resolved.